pkg/openshift/template.go

```diff
@@ -25,25 +25,33 @@ const letterBytes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456
 
 func init() {
 	PackagedTemplates["cloudapp"] = templates.CloudAppTemplate
+
 	PackagedTemplates["cache-redis"] = templates.CacheRedisTemplate
 	PackagedTemplates["data-mongo"] = templates.DataMongoTemplate
+	PackagedTemplates["data-mongo-replica"] = templates.DataMongoReplicaTemplate
 	PackagedTemplates["data-mongo-job"] = templates.DataMongoConfigJob
 	PackagedTemplates["data-mysql"] = templates.DataMySQLTemplate
+
 }
 
 func (tl *templateLoaderDecoder) Load(name string) (*template.Template, error) {
 
 	var t = template.New("")
+	var genPass = func() string {
+		b := make([]byte, 16)
+		for i := range b {
+			b[i] = letterBytes[rand.Intn(len(letterBytes))]
+		}
+		return string(b)
+	}
+	var generatedPass = genPass() // a single generated password per template load
```

Comment:
Can we allow for more than one if required? I could imagine a service needing a user and an admin password.